static/styles/left_sidebar.css

```diff
@@ -478,16 +508,6 @@ li.expanded_private_message {
     }
 }
 
-.show-all-streams {
-    a {
-        color: hsl(0, 0%, 20%);
-    }
-
-    .fa-chevron-left {
-        text-decoration: none;
-    }
-}
```

Comment:
I think you should extract any commits that intend to refactor the existing CSS for "more topics" into independent, correct preparatory commits; it's very hard to figure out where this code ends up.

Reply:
Done in #21726